### DISCUSSION TITLE
Question about compiling code under Linux system.

### DISCUSSION THREAD

**Discussion by Aping1100** [Anura3D/Anura3D_OpenSource#67] [Q&A] [answered]
Hi everyone
Recently I have to conduct calculations under Linux system, but I have no idea to compile Aunra3D code under Linux. Is there any suggestion or tutorial for this situation?  Hope for your answers.

**Comment by murphy-john** [comment, marked as answer]
Hello @Aping1100, we don't have support for compiling the code in Linux at this time. A couple members of the community are working on trying to figure out how to do this. We don't have an estimate for how long it will take.

**Reply by ayerrocolom** [reply]
The key issue when trying to compile Anura3D in Linux is the external soil model that is called as a dll.

**Reply by Aping1100** [reply]
Thanks for the answers.

**Reply by alansill** [reply]
We are getting requests for this too. Since VS and OneAPI, as well as Paraview, are available in Linux, can we request that this be escalated?
